src/components/NcSelect/NcSelect.vue

```diff
@@ -945,6 +945,9 @@ body {
 	--vs-state-disabled-color: var(--color-text-maxcontrast);
 	--vs-state-disabled-controls-color: var(--color-text-maxcontrast);
 	--vs-state-disabled-cursor: not-allowed;
+	--vs-disabled-bg: var(--color-background-dark);
+	--vs-disabled-color: var(--color-text-maxcontrast);
+	--vs-disabled-cursor: not-allowed;
```

Comment:
🤔 the `--vs-state-disabled-*` variables are already set above and are aliased to these variables internally https://github.com/nextcloud-deps/vue-select/blob/9294fdcbf39380fd5cc766144b18b6f86cf107f4/src/css/global/states.css#L13C35-L15

Is this solvable without adding non-publicly documented variables https://vue-select.org/guide/css.html#available-css-variables-3-18?

Reply:
I have no clue why, but it fixes it. As per https://github.com/nextcloud/nextcloud-vue/pull/4079#discussion_r1189544610 it could be the order, since `--vs-disabled-bg: var(--vs-state-disabled-bg);` is not a permanent redirect, but it writes the currently value of the second var into the first one. So maybe we only replace the `--vs-state-disabled-bg` after that and therefore also need to overwrite the first var

Reply:
Or well:
https://github.com/nextcloud/nextcloud-vue/blob/93f9ab752f7d64dd14afbe6d549f0e1fab01d6ae/src/components/NcSelect/NcSelect.vue#L525
I'm pretty sure that is exactly what is going on

Reply:
I think this happens because:
`--vs-disabled-bg: var(--vs-state-disabled-bg);`  from vue-select if on the `:root` selector.

And our `--vs-state-disabled-bg: var(--color-background-dark);` is on the `body` selector.

So when `--vs-disabled-bg` is set, the `--vs-state-disabled-bg` variable holds the old value.

Reply:
So I think it has the same cause as this one: https://github.com/nextcloud/server/pull/36462